server/controllers/configurationsController/index.js

```diff
@@ -40,11 +43,10 @@ const ConfigurationsController = {
     try {
       const { appDb } = req.app.locals
       const { config_id } = req.params
-      const updatedConfiguration = await ConfigModel.update(
-        appDb,
-        config_id,
-        req.body
-      )
+      const updatedConfiguration = await ConfigModel.update(appDb, config_id, {
+        ...req.body,
+        updatedAt: new Date().toISOString(),
```

Comment:
I can't answer for created because that's from before we started working on the codebase, in other pieces of code our implementation is ISO string as updatedAt is new i'm following that

Reply:
we should have consistent timestamp formats, one way or the other.